my_app.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask, jsonify, request
+from werkzeug.exceptions import abort
+from clarifai.rest import ClarifaiApp
+"""Collect file names and description from Internet"""
+
+my_app = Flask(__name__)
+
+
+FILES = []
+
+api_key = '4756270c660f4c3fb9ad13cf24d70588'
```

Comment:
For homework and study is ok. But be accurate with secret and access keys in your git repo.

Reply:
Yes, you are right.

Reply:
Please check corrected file.